specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,689 @@
+            "name": "$filter",
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCFList"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ManagedCCFList": {
+            "$ref": "./examples/ManagedCCF_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ConfidentialLedger/managedCCFs/": {
+      "get": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Retrieves information about all Managed CCF resources under the given subscription",
+        "description": "Retrieves the properties of all Managed CCF.",
+        "operationId": "ManagedCCF_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "type": "string",
+            "description": "The filter to apply on the list operation. eg. $filter=ledgerType eq 'Public'",
+            "name": "$filter",
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; response contains the list of Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCFList"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ManagedCCFListBySub": {
+            "$ref": "./examples/ManagedCCF_ListBySub.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ConfidentialLedger/managedCCFs/{appName}/backup": {
+      "post": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Performs the backup operation on a Managed CCF Resource.",
```

Comment:
Consider not abbreviating CCF in docs.

Reply:
Our documents do specify what CCF (Confidential Consortium Framework) stands for so users creating a Managed CCF resource should be aware of the resource as a whole. Since the resource itself is Managed CCF, it should be fine to reference the resource as Managed CCF in our docs since we do have sections explaining the meaning behind the entire service name.